### ISSUE TITLE
json5 latest is now 1.0.2 on npmjs - intentional?

### ISSUE THREAD

**Issue by sverweij** [json5/json5#299]
The _latest_ version of json5 from npmjs is [1.0.2 since yesterday](https://www.npmjs.com/package/json5?activeTab=versions). Is that intentional? 

As a consequence tooling that has a policy to upgrade to any latest version is advising to 'upgrade' from version ^2 to 1.0.2 - e.g. with `npm outdated`:

```sh
 ~/things $ npm outdated                         
Package            Current  Wanted  Latest  Location                        Depended by
json5                2.2.2   2.2.2   1.0.2  node_modules/json5              things
```

> If it's not intentional I believe the only recourse is to re-publish 2.2.2 as version 2.2.3 (with the default tag/ tag 'latest'). It could be useful to also re-publish 1.0.2 as 1.0.3, but with tag v1 or something similar.

**Comment by jordanbtucker** [comment]
Thanks for alerting me to this. It was not intentional. I'll get that fixed soon.

**Comment by jordanbtucker** [comment]
Fixed in v2.2.3!

**Comment by jordanbtucker** [comment]
I was able to add a `previous` tag to v1.0.2 without having to release v1.0.3. I was not able to use `v1` as the tag name.

> **Note:** Since dist-tags share a namespace with semantic versions, avoid dist-tags that conflict with existing version numbers. We recommend avoiding dist-tags that start with a number or the letter "v".
> - [Adding dist-tags to packages](https://docs.npmjs.com/adding-dist-tags-to-packages)
